src/Angor/Client/Pages/Settings.razor

```diff
@@ -126,9 +126,7 @@
                                                     @foreach (var feature in _featureFlags)
                                                     {
                                                         <div class="form-check mb-3">
-                                                            <input class="form-check-input"
-                                                                type="checkbox"
-                                                                id="@feature.Key"
+                                                            <input class="form-check-input" type="checkbox" id="@feature.Key"
```

Comment:
you have changes that are unrelated, no need to make such changes, the only changes in the ui should be the additional section